src/main/java/slimeknights/tconstruct/tables/inventory/table/toolstation/ToolStationOutSlot.java

```diff
@@ -0,0 +1,34 @@
+package slimeknights.tconstruct.tables.inventory.table.toolstation;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.inventory.CraftResultInventory;
+import net.minecraft.inventory.container.Slot;
+import net.minecraft.item.ItemStack;
+
+import javax.annotation.Nonnull;
+
+public class ToolStationOutSlot extends Slot {
+
+  public ToolStationContainer parent;
+
+  public ToolStationOutSlot(int index, int xPosition, int yPosition, ToolStationContainer parent) {
+    super(new CraftResultInventory(), index, xPosition, yPosition);
+
+    this.parent = parent;
+  }
+
+  @Override
+  public boolean isItemValid(ItemStack stack) {
+    return false;
+  }
+
+  @Nonnull
```

Comment:
For the record, all our stuff is `@Nonnull` by default, no need to manually add the annotation